appveyor.yml

```diff
@@ -0,0 +1,28 @@
+image: ubuntu1804
+stack: node 8
+build: off
+
+cache:
+  - node_modules -> package.json
+
+install:
+  - sudo apt-get update
+  - sudo apt-get -fy install composer
+  - npm install -g gulp
+  - npm install
+
+before_build:
+  - VERSION="${APPVEYOR_REPO_TAG_NAME:-dev-$APPVEYOR_REPO_BRANCH}"
+  - ARTIFACT="municipio-${VERSION//\//__}.zip"
+
+build_script:
+  - composer config version "$VERSION"
+  - truncate -s 0 vendor/autoload.php
+  - rm -rf assets/dist
+  - gulp build
```

Comment:
If you follow my above suggestion, then this changes from "gulp build" to "npm run build"
And if removing the assets/dist folder is required for it, how about putting that as a "prebuild" script to make sure it gets run?

Reply:
Certainly!

Reply:
@jofo8948 I'll make a note to make a separate PR to Municipio. We may also want to look into updating their dependencies. For this particular PR we'll just use `npx` though.